docs/samples/browser-plugin-meetings/app.js

```diff
@@ -2129,6 +2155,8 @@ function addMedia() {
     console.error(error);
   });
 
+  handleStreamPublishedState(meeting);
```

Comment:
we need to unregister these listeners when leaving the meeting, otherwise a 2nd copy of listeners will be registered when user joins the same meeting a 2nd time. Maybe to make it simpler we could just call `handleStreamPublishedState()` only once when meeting is created instead of here in addMedia()?

Reply:
Will update it as suggested.

Reply:
Discussed with Marcin and it makes sense to not unregister it.